Postgresql/legos/postgresql_check_active_connections/postgresql_check_active_connections.py

```diff
@@ -0,0 +1,66 @@
+##
+# Copyright (c) 2023 unSkript, Inc
+# All rights reserved.
+##
+from typing import Optional, Tuple
+from pydantic import BaseModel, Field
+from tabulate import tabulate
+
+
+class InputSchema(BaseModel):
+    threshold_connections: Optional[int] = Field(
+        100,
+        description='Number of connections to consider as the threshold.',
+        title='Threshold no. of connections',
+    )
+
+
+def postgresql_check_active_connections_printer(output):
+    status, data = output
+
+    if not status and data:
+        headers = ["Active Connections"]
+        table_data = [[record["active_connections"]] for record in data]
+        print(tabulate(table_data, headers=headers, tablefmt="grid"))
+    else:
+        print("Active connections are below the threshold.")
+
+
+def postgresql_check_active_connections(handle, threshold_connections: int = 100) -> Tuple:
```

Comment:
I would change this a bit - the thresholding should be on a percentage. Eg more than 85% is a failure.
100 is the default pool count, but we should be reading the current value of the configured pool count.

ie we are checking if the pool is at more than 85% utilization.

Reply:
Done..